velox/exec/Window.cpp

```diff
@@ -88,8 +102,8 @@ Window::Window(
   createWindowFunctions(windowNode, inputType);
 }
 
-void Window::createWindowFunctions(
-    const std::shared_ptr<const core::WindowNode>& windowNode,
+Window::WindowFrame Window::createWindowFrame(
+    core::WindowNode::Frame frame,
     const RowTypePtr& inputType) {
   auto constantArg = [&](const core::TypedExprPtr arg) -> const VectorPtr {
```

Comment:
This lambda is repeated in two places. Lets create a simple function called createConstantForWindowArg(...) and reuse.